src/components/transactions/Warnings/MarketWarning.tsx

```diff
@@ -0,0 +1,55 @@
+import { Trans } from '@lingui/macro';
+import { Link, Typography } from '@mui/material';
+
+import { Warning } from '../../primitives/Warning';
+
+const WarningMessage = ({ market }: { market: string }) => {
+  if (market === 'Harmony') {
+    return (
+      <Trans>
+        Due to the Horizon bridge exploit, certain assets on the Harmony network are not at parity
+        with Ethereum, which affects the Aave V3 Harmony market.
+      </Trans>
+    );
+  } else if (market === 'Fantom') {
+    return <Trans>Per the community, the Fantom market has been frozen.</Trans>;
+  } else {
+    return <></>;
+  }
+};
+
+const getLink = (market: string, forum: boolean | undefined): string => {
+  if (market === 'Harmony') {
+    if (forum) {
+      return 'https://governance.aave.com/t/harmony-horizon-bridge-exploit-consequences-to-aave-v3-harmony/8614';
+    } else {
+      return 'https://snapshot.org/#/aave.eth/proposal/0x81a78109941e5e0ac6cb5ebf82597c839c20ad6821a8c3ff063dba39032533d4';
+    }
+  } else if (market === 'Fantom') {
+    if (forum) {
+      return 'https://governance.aave.com/t/arc-aave-v3-fantom-freeze-reserves/9166';
+    } else {
+      return 'https://snapshot.org/#/aave.eth/proposal/0xeefcd76e523391a14cfd0a79b531ea0a3faf0eb4a058e255fac13a2d224cc647';
```

Comment:
Similarly.  I think we should create a `src/constants/` directory for this.  This is how I've typically seen it in large frontend apps, modularizing the constants.  Was thinking about this as I was writing `CHART_HEIGHT`s this week for the graphs.

Reply:
Also will be covered in other comment ^

Reply:
Once we get that file setup, we could setup a mapping of market name to proposal info. Then I think in the asset list components we wouldn't need to do any checking of market name, we just look up in the mapping by market name, if it's empty don't render. Could help clean up some of those if statements in the other pages.

Reply:
I like this idea